src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs

```diff
@@ -53,25 +53,28 @@ public sealed partial class Utf8JsonWriter : IDisposable, IAsyncDisposable
         private BitStack _bitStack;
 
         /// <summary>
-        /// This 4-byte array stores the partial code point leftover when writing a string value
-        /// segment that is split across multiple write calls. The first 3 bytes provide space
-        /// to store the leftover bytes using the source encoding and the last byte is the number
-        /// of bytes used to store the partial code point.
+        /// This 3-byte array stores the partial code point leftover when writing a string value
+        /// segment that is split across multiple write calls.
         /// </summary>
 #if !NET
         private byte[]? _partialCodePoint;
-        private Span<byte> PartialCodePointRaw => _partialCodePoint ??= new byte[4];
+        private Span<byte> PartialCodePointRaw => _partialCodePoint ??= new byte[3];
```

Comment:
Looks like this property can be lifted out of the build conditional now?

Reply:
This still needs a conditional since the full .NET uses an inline array instead of byte[] so that doesn't need the lazy initialization. I can move it out, make the statement multiline and just conditional the null coalescing assignment, but I don't know how much cleaner that is than just leaving it next to the underlying field.